package.json

```diff
@@ -342,6 +342,7 @@
     "url": "https://github.com/Samsung/ONE-vscode.git"
   },
   "dependencies": {
+    "@typescript-eslint/typescript-estree": "^5.29.0",
```

Comment:
Q. Why is it necessary?

Reply:
I will check this. I think this comes somewhat differ place.....:(